tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle@2023_01_11/callee_guaranteed.pysnap

```diff
@@ -141,25 +141,25 @@ system:
           frame (ignored by stack trace rule (category:threadbase -group v-group))
             function*
               "UIApplicationMain"
-          frame (ignored by stack trace rule (category:threadbase -group v-group))
+          frame*
             function*
               "-[UIApplication _run]"
```

Comment:
The hash changes because now this frame is being considered for grouping (the ones below are considered `internals`).

@mitsuhiko @untitaker should we also add this rule? `family:native package:UIKitCore function:-[UIApplication _run] category=internals`

```
{
              "function": "-[UIApplication _run]",
              "package": "/System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore",
              "in_app": false,
              "data": {}
            },
```

Reply:
I guess yes.

Reply:
If I [add it](https://github.com/getsentry/sentry/pull/61685/files#diff-0ac6225892ddf6f49368585496603f476c9d766cabb9a42ce4d84ba30e4e334bR48), it will also generate new groups because in some cases it was being considered for grouping.